i18n/ptb/package.i18n.json

```diff
@@ -36,5 +36,13 @@
 	"pacCLI.envAndSolutionsPanel.copyVersionNumber.title": "Copiar Número da Versão",
 	"microsoft-powerapps-portals.preview-show.title": "Portal do PowerApps -> Mostrar versão preliminar",
 	"microsoft-powerapps-portals.webExtension.init.title": "Inicializar Extensão da Web",
-	"microsoft-powerapps-portals.webExtension.limitation": "Não há suporte para PAC CLI no VS Code para a Web"
+	"microsoft-powerapps-portals.webExtension.limitation": "Não há suporte para PAC CLI no VS Code para a Web",
+	"microsoft-powerapps-portals.walkthrough.title": "Editar código do Power Pages",
+	"microsoft-powerapps-portals.walkthrough.description": "Descubra como você pode criar sites completos e modernos, diretamente no VS Code",
+	"microsoft-powerapps-portals.walkthrough.overview.title": "Visão Geral",
+	"microsoft-powerapps-portals.walkthrough.overview.description": "A extensão do Power Pages permite um sistema de arquivos personalizado que é sincronizado com a estrutura do projeto do estúdio do Power Pages.\n \n A visualização está vinculada ao Power Pages e permite que você veja suas alterações.\n[Saiba Mais] (command:powerplatform-walkthrough.overview-learn-more)",
```

Comment:
No spaces permitted between the `]` and `(` in the markdown link format.   
(3 in this file)